### ISSUE TITLE
v5.205: sending mail using CRAM-MD5 does not work anymore

### ISSUE THREAD

**Issue by gen2** [thunderbird/thunderbird-android#2310]
Yesterday i updated my device using the f-droid repo. So i installed the v5.205 version of K9 mail. After the update sending mail on the accounts of my server@home is not working any more on my tablet (the iPhone using the same settings still works). And since i'm on holiday at the other end of the world a change on the server can be excluded. K9 mail reports there is no such auth method:

K9-Mail version: 5.205
Device make: asus
Device model: Nexus 7
Android version: 6.0.1

com.fsck.k9.mail.MessagingException: Authentication method CRAM-MD5 is unavailable.
 at com.fsck.k9.mail.transport.SmtpTransport.open(SmtpTransport.java:340)
 at com.fsck.k9.mail.transport.SmtpTransport.sendMessageTo(SmtpTransport.java:504)
 at com.fsck.k9.mail.transport.SmtpTransport.sendMessage(SmtpTransport.java:496)
 at com.fsck.k9.controller.MessagingController.sendPendingMessagesSynchronous(MessagingController.java:3014)
 at com.fsck.k9.controller.MessagingController.access$1700(MessagingController.java:115)
 at com.fsck.k9.controller.MessagingController$21.run(MessagingController.java:2895)
 at com.fsck.k9.controller.MessagingController.runInBackground(MessagingController.java:204)
 at com.fsck.k9.controller.MessagingController.access$000(MessagingController.java:115)
 at com.fsck.k9.controller.MessagingController$1.run(MessagingController.java:173)
 at java.lang.Thread.run(Thread.java:818)K-9 Mail version: v5.205

But this is simply wrong, and i checked it using telnet to port 25 of my server (ok, it's not the ssl port, but i tried to configure K9 mail to use the non ssl 25 port and runned into the same issue).

Android version: android 6.0.1 / Cyanogen 13.0

Account type (IMAP, POP3, WebDAV/Exchange): cyrus imap server, sendmail on ssl port using CRAM-MD5 authentication.

Since i'm on holiday i cannot provide adb debug output by now - sorry.

**Comment by cketti** [comment]
You didn't include which version is still working. There have been no changes to the SMTP code since 5.200 was released.

Without a log of the communication there's also not much we can do. Closing this issue until logs are provided.

**Comment by Valodim** [comment]
This error occurs when CRAM-MD5 is selected as authentication method when the servers says it's not available. So yes, it's very likely that something changed on the server. Try changing your auth type from "Encrypted password" to "Normal password"

**Comment by gen2** [comment]
@cketti v5.203 works, i downgraded and tested today

@Valodim i do strangs things during my holidays, but i don't update my mail server (if not really necessary) beeing 15000km away from the console ;) Seriously, i checked it using telent, the CRAM-MD5 option is there, and a try "succeded" by offering some base64 encoded string to me (the client)

I'll provide a debug log next week or so - i've no desktop like system handy right now.

**Comment by Valodim** [comment]
Hm, can't argue with that ;)

Debug log will be super helpful, thanks.

**Comment by Helaine-Gretz** [comment]
I've just installed K9 on a new phone.
Phone is Nokia 8 running Android 8.1.0
Version of K9 is 5.4.0.3

I tried to set up a Countermail account as per this tutorial https://support.countermail.com/kb/faq.php?id=101 but it isn't possible to select CRAM-MD5 as an option at step 5 because the option simply doesn't appear in the authentication type drop down menu. If I select "encrypted password" instead I get the following error 

Username or password incorrect. (Command: AUTHENTICATE CRAM-MD5; response: #1# [NO, [AUTHENTICATIONFAILED], Authentication failed.])
